### PR TITLE
Show all exercise info

### PR DESCRIPTION
**Pull request by kawerewagaba**
### What does this PR do?
This PR shows all infos for an exercise.

### Description of Task to be completed?
Create a new read-only API endpoint that collects all the information of an exercise such as muscles, category, images, etc.

### How should this be manually tested?
Test locally by navigating to this [url]: ```http://127.0.0.1:8000/api/v2/exerciseinfo/```

### Any background context you want to provide?
Essentially, this makes it easier for clients to get all the infos since only a single request is needed to be done. Saves time and resources.

### What are the relevant pivotal tracker stories?
PT Story ID: [152919317](https://www.pivotaltracker.com/story/show/152919317)

### Some screenshot:
screen shot 2017-12-06 at 12 10 21
screen shot 2017-12-06 at 12 09 34

### CODE REVIEW

**Comment by Sharonsyra** [comment]
@ckwagaba merge conflicts.

**Comment by Sharonsyra** [comment]
@ckwagaba improve the structure of your PR. It needs to have different sections to it and at the same time explaining how to test it.

**Comment by Sharonsyra** [comment]
@ckwagaba yet to see feedback implemented on this.

**Comment by kawerewagaba** [comment]
On it...

**Comment by kawerewagaba** [comment]
Hey, @Sharonsyra! Please review my PR.
Thanks.

**Review by jonathankamau — CHANGES_REQUESTED** [review]
You have conflicts on your PR. Please resolve.

**Review by jonathankamau — CHANGES_REQUESTED** [review]
You still have conflicts on this branch. Please resolve.

**Review by jonathankamau — CHANGES_REQUESTED** [review]
Make the PT ID a link as per the guidelines I have posted on the team channel.

**Comment by kawerewagaba** [comment]
PT ID made link. Resolving conflict...

**Review by jonathankamau — CHANGES_REQUESTED** [review]
Add the PT ID to your PR title so it appears like this: `#152919317 Show all exercise info`

**Comment by kawerewagaba** [comment]
@Sharonsyra and @jonathankamau I think I've made the changes. Please review.
Thanks

**Review by jonathankamau — CHANGES_REQUESTED** [review]
The PR title still needs to be changed. I had posted this: `Add the PT ID to your PR title so it appears like this: '#152919317 Show all exercise info'`

File: settings.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 
+
```
Comment (jonathankamau) [inline settings.py:2]: You have two empty lines here.

**Pull request merged**